### PR TITLE
Side effects handled

### PR DESCRIPTION
**Pull request by Jalitha99**
[ITP-1188]
[ITP-1176]
[ITP-1158]

[ITP-1188]: https://fcodelabs.atlassian.net/browse/ITP-1188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ITP-1176]: https://fcodelabs.atlassian.net/browse/ITP-1176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ITP-1158]: https://fcodelabs.atlassian.net/browse/ITP-1158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+        <Routes>
```
Comment (udara-sinister) [inline src/App.tsx]: Use a separate component to handle the routing

File: src/Firebase.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getFirestore } from "firebase/firestore";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+const firebaseConfig = {
```
Comment (udara-sinister) [inline src/Firebase.ts:8]: Use a separate .env file to store sensitive information

File: src/pages/DiaryHome/home.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    const [title, setSubmitNew] = useState('');
+    const [description, setDescription] = useState('');
+    const [collapseValue, setCollapse] = useState(false)
+    // const [cardContent, setCardContent] = useState([]);
+    const params = useParams();
+    const userName = params.userName;
+    const dispatch = useDispatch(); 
+
+    // const readData = async() =>{
+    //   var cardContent = [] as any;
```
Comment (udara-sinister) [inline src/pages/DiaryHome/home.tsx]: Remove unnecessary comments

File: src/pages/DiaryHome/home.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      // useEffect(() => {
+      //   readData();
+      // }, [isRender]);
+
+    const handleSubmit = async (e : any) => {
+        e.preventDefault();
+
+        // Add a new document in collection "Card"
+        // try {
+        //     if(title || description){
```
Comment (udara-sinister) [inline src/pages/DiaryHome/home.tsx]: remove unnecessary comments

**Review by udara-sinister — CHANGES_REQUESTED** [review]
There are several unnecessary comments in the source code. I have mentioned some of them and there are some more of them. Make sure to remove those comments before the next pr

File: src/Firebase.ts (hunk 2 above)
Comment (Jalitha99) [reply src/Firebase.ts:8]: added a separate.env file

**Review by Jalitha99 — COMMENTED** [review]
Issues resolved

**Pull request closed without merging**